test/chi/components/avatar.pug

```diff
@@ -73,3 +73,16 @@ each type in types
                           [`-transparent`]: transparent
                         }
                       ) AA
+
+                    else if type === 'avatar-group'
+                      div.chi-avatar-group
+                        div(
+                          class={
+                            "chi-avatar": true,
+                            [`-${color}`]: type !== 'default',
+                            [`-${size}`]: style !== 'default',
+                            [`-transparent`]: transparent
+                          }
+                        ) L 
+                        | Label&nbsp;
+                        span=  color
```

Comment:
@sabah1910 there's an issue when this markup renders in the Html. It is not rendering as on the example page. Attached is a screenshot. Can you solve it?

<img width="400" alt="Screen Shot 2020-07-22 at 12 25 52 PM" src="https://user-images.githubusercontent.com/17043903/88166189-2b3db580-cc17-11ea-8b62-a0680bd48203.png">

Reply:
@jllr  When I type color next to Label
![image](https://user-images.githubusercontent.com/58184901/88213030-6d65f780-cc15-11ea-8398-2fad50ebb067.png)
It displays this on the test
![image](https://user-images.githubusercontent.com/58184901/88213105-85d61200-cc15-11ea-8bae-f123b030011c.png)
Please advise

Reply:
@sabah1910 please delete line 88 span= color and update line 87 with | Label #{color}

Reply:
@jllr Done